src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -0,0 +1,55 @@
+package mate.academy.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinTable;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+
+@Entity
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToMany
+    @JoinTable(name = "cart_tickets")
```

Comment:
```suggestion
    @JoinTable(name = "cart_tickets",
     joinColumns =
            @JoinColumn(name = "shopping_cart_id"),
            inverseJoinColumns =
            @JoinColumn(name = "ticket_id"))
```

we can rename columns as well